.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+
+node_js:
+  - '10.7'
+
+cache:
+  directories:
+    - node_modules
+
+matrix:
+  fast_finish: true
+
+install:
+  - npm install
+  - npm install -g codecov
+
+env:
+  - PACKAGE=superset-ui-core
+
+script:
+  - cd ./packages/$PACKAGE && yarn install && yarn run lint && yarn run test
```

Comment:
You can split the script to br a series of commands, 
```
script:
  - cd ./packages/$PACKAGE
  - yarn install
  - ...
```

Reply:
that's prob nicer for debugging too 👍